cypress/support/pages/ContactUsPage.js

```diff
@@ -0,0 +1,59 @@
+var emailLocator = 'input[id="email"]';
+var orderReferenceLocator = 'input[id="id_order"]';
+var messageLocator = 'textarea[id="message"]';
+var submitMessageLocator = 'button[id="submitMessage"]';
+var pageURL = 'http://automationpractice.com/index.php?controller=contact';
+var email = 'email@email.com'
```

Comment:
Nice to see the test data called out here rather than in the steps.

Could have gone one step further and implemented something like Faker to generate test data on the fly...but this is a good enough approach as it.